From 85fcd50b3f33cb811f4ba79a53b3a6b29054c721 Mon Sep 17 00:00:00 2001
From: Ryan Lortie <desrt@desrt.ca>
Date: Tue, 17 Jan 2012 15:55:50 +0000
Subject: Revert "set the G_PARAM_CONSTRUCT flag on the GtkWindow:application property"

This reverts commit d4fe912879ce1b19490ba5729f67a27b1cf397c9.

This patch caused some unanticipated compatibility issues.
---
diff --git a/gtk/gtkwindow.c b/gtk/gtkwindow.c
index 62b9ac0..93eb4a7 100644
--- a/gtk/gtkwindow.c
+++ b/gtk/gtkwindow.c
@@ -1044,7 +1044,7 @@ gtk_window_class_init (GtkWindowClass *klass)
                                                         P_("GtkApplication"),
                                                         P_("The GtkApplication for the window"),
                                                         GTK_TYPE_APPLICATION,
-                                                        GTK_PARAM_READWRITE | G_PARAM_CONSTRUCT | G_PARAM_STATIC_STRINGS));
+                                                        GTK_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
   window_signals[SET_FOCUS] =
     g_signal_new (I_("set-focus"),
--
cgit v0.9.0.2
